cobalt/site/docs/gen/cobalt/doc/cvals.md

```diff
@@ -79,7 +79,7 @@ Here are examples of its usage:
 
 #### PublicCVals
 
-*   **Count.DOM.EventListeners** - The total number of EventListeners in
```

Comment:
Ugh. This is a breaking change to any client code that relies on them.

Reply:
Internally tracked at b/278899927 - i think that needs to be reopened.

Reply:
Resolving here. Following up internally